config/examples/backend_redis.yml

```diff
@@ -1,29 +1,18 @@
-# Configuration for Redis. Separate per each environment.
-#
-# Accepted options are:
-#
-# servers       list of servers (hostname:port) to connect to. The first one is the master, the
-#               others are used for failover (no sharding yet).
-# db            database to use. A number from 0 to 10 (default is 0)
-# timeout
-# password
-# thread_safe
-base: &base
-  url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/0') %>"
+base: &default
+  url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/6') %>"
```

Comment:
The URL has change slightly, is it intentional?

Reply:
Yes, this is intentional, and the reason is that [porta-dev-tools](https://github.com/3scale-labs/porta-dev-tools/blob/master/bin/porta#L705) uses DB